x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/HashAggregationOperatorTests.java

```diff
@@ -96,4 +101,154 @@ protected void assertSimpleOutput(List<Page> input, List<Page> results) {
             max.assertSimpleGroup(input, maxs, i, group);
         }
     }
+
+    public void testTopNNullsLast() {
```

Comment:
These tests are quite shallow, but as the BlockHash is also tested, and no Operator changes were made, I think this is fine for now (?).
When we link the new TopNBlockHash with the language side, I'll review this again

Reply:
:+1:

I could see us doing a lot more randomized fun to this, out of paranoia.